purchases/src/main/kotlin/com/revenuecat/purchases/models/SubscriptionOption.kt

```diff
@@ -93,4 +93,11 @@ interface SubscriptionOption {
         }
 
     val purchasingData: PurchasingData
+
+    /**
+     * For installment subscriptions, the details of the installment plan the customer commits to.
+     * Null for non-installment subscriptions.
+     * Installment plans are only available for Google Play subscriptions.
+     */
+    val installmentsInfo: InstallmentsInfo?
```

Comment:
I considered exposing this as part of the `StoreProduct` instead of the `SubscriptionOption`... Currently that should work, since a base plan can only have installment plans, but this matched better our current architecture. We can also expose a similar field in `StoreProduct` default to the default subscription option, but I thought to leave it at this level for now. 

Lmk if you feel this belongs at the `StoreProduct` level.